CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## 0.10.1dev
+* [Feature] Automatically connect if the `dsn_filename` contains a `default` section
```

Comment:
please add that it looks in the default location `~/.jupysql/connections.ini`

Reply:
https://github.com/ploomber/jupysql/pull/819/commits/7c2ec814c8e6b4534de137a46e80a2c4ca6f6b49